src/parser/monk/mistweaver/modules/talents/Tier45Comparison.tsx

```diff
@@ -60,20 +70,24 @@ class Tier45Comparison extends Analyzer {
   talents = [this.manatea, this.lifecycles, this.sotc];
 
   returnedFromSelected = 0;
-  best;
+  best: BestTalent = {
```

Comment:
`best!: BestTalent;`

or

`best: BestTalent | null = null;` (and then check for null when its accessed in the suggestions)